Makefile

```diff
@@ -157,6 +157,10 @@ ifeq ($(V),0)
   Q=@
 endif
 
+ifeq ($(NO_MPU),1)
+  CFLAGS+=-DWOLFBOOT_NO_MPU
```

Comment:
What about people who are building wolfBoot not using our Makefile? If they are on ARM it will default to having MPU code enabled. Perhaps it should be reverse logic, so it's disabled by default?

Reply:
This is done on purpose so MPU is on by default, unless explicitly disabled (when compiling without the makefile, same applies as other options with the same logic such as NO_VTOR).

It is safe to keep MPU on on platforms that don't support it, because the code will do nothing if MPU->TYPE is '0'. (MPU->TYPE must be implemented as '0' by ARM Cortex-M cores when the feature is not available).